### DISCUSSION TITLE
Could we add vibration motors to the gloves for extra feedback?

### DISCUSSION THREAD

**Discussion by leappad** [LucidVR/opengloves-driver#182] [Ideas]
I realized it has haptic feedback by allowing your hands to wrap around virtual objects but would it be possible to use the valve index emulation to add vibration haptic motors like this [https://www.amazon.com/dp/B073YFR5WR](https://www.amazon.com/dp/B073YFR5WR) to the gloves? I think tools like ovr advanced toolkit could turn now the power of the haptics so it could add some extra immersion and you could tweak the power of it

**Comment by danwillm** [comment]
Yes, this is definitely possible - while it's not currently implemented, it is reasonably trivial to do so, and I'm waiting on a valid use case/design someone has before adding it in. If you have that project, let me know and I can add it in.

**Comment by danwillm** [comment]
Currently on this pr: https://github.com/LucidVR/opengloves-driver/pull/187
will update steam soon
